.cargo/config.toml

```diff
@@ -0,0 +1,17 @@
+[net]
+git-fetch-with-cli = true
+
+[patch.crates-io]
+embedded-hal = { git = "https://github.com/Dirbaio/embedded-hal", rev = "d90b59b0f48bba1e65129d9f8d5c702d47794d9f"}
```

Comment:
patching embedded-hal should no longer be needed.

For embassy crates I'd keep them in the individual `Cargo.toml` files, it makes it easier to copy one example project out of the repo because it's self contained, but no strong opinion from me, if you prefer centralizing them that's fine too.

Reply:
I feel like it's easier to have them all in the same place, prevents having redundant patches imo